travis-checks/Dockerfile

```diff
@@ -14,7 +14,9 @@ RUN apt-get install -y \
     autoconf \
     automake \
     make \
-    cmake
+    cmake \
+    clang-format \
+    git
```

Comment:
You've added them to `for 3rd-parties` section, but that's not quite correct. Let's introduce a new section, say `for scons fmt`.